.circleci/common.sh

```diff
@@ -105,7 +105,7 @@ function build_torch_xla() {
   XLA_DIR=$1
   pushd "$XLA_DIR"
   python setup.py install
-  pip install plugins/cuda -v
+  # pip install plugins/cuda -v
```

Comment:
let's leave a comment about this commenting out :)

Reply:
done